### PR TITLE
Fix deep input

### PR DESCRIPTION
**Pull request by jderusse**
When a request contains sub-objects the return requestBody was wrong.

Added a test: src/Sqs/Tests/Unit/Input/SendMessageRequestTest.php to assert output

### CODE REVIEW

File: src/Ses/Input/SendEmailRequest.php [hunk 1]
```diff
@@ -127,26 +127,104 @@ public function getReplyToAddresses(): array
...
+            foreach ($input as $value) {
+                ++$indices->k7c3bed1;
+                (static function ($input) use (&$payload, $indices) {
+                    $payload["EmailTags.{$indices->k7c3bed1}.Name"] = $input->getName();
+                    $payload["EmailTags.{$indices->k7c3bed1}.Value"] = $input->getValue();
+                })($value);
+            }
+        })($this->EmailTags);
+
+        if (null !== $v = $this->ConfigurationSetName) {
```
Comment (Nyholm) [inline src/Ses/Input/SendEmailRequest.php]: Oh, This code is not super pretty.

File: src/Sqs/Input/CreateQueueRequest.php [hunk 2]
```diff
@@ -69,15 +69,24 @@ public function gettags(): array
...
-        if (null !== $this->QueueName) {
-            $payload['QueueName'] = $this->QueueName;
-        }
-        if (null !== $this->Attributes) {
-            $payload['Attribute'] = $this->Attributes;
-        }
-        if (null !== $this->tags) {
-            $payload['Tag'] = $this->tags;
-        }
+        $indices = (object) [];
```
Comment (Nyholm) [inline src/Sqs/Input/CreateQueueRequest.php]: What about? 
`$indices = new \stdObject()`

Isnt this the same?

File: src/Ses/Input/SendEmailRequest.php (hunk 1 above)
Comment (jderusse) [reply src/Ses/Input/SendEmailRequest.php]: Agreed :/

I'm open to any suggestion

File: src/Sqs/Input/CreateQueueRequest.php (hunk 2 above)
Comment (jderusse) [reply src/Sqs/Input/CreateQueueRequest.php]: fixed

File: src/Ses/Input/SendEmailRequest.php (hunk 1 above)
Comment (Nyholm) [reply src/Ses/Input/SendEmailRequest.php]: I don’t have any...
Let’s merge and improve this later.

**Pull request merged**